docs/Installation.md

```diff
@@ -41,6 +41,7 @@ The following presents an incomplete list:
 LF_WORKER={SCION,IPV4,FWD}
 LF_DRKEY_FETCHER={SCION,MOCK}
 LF_LOG_DP_LEVEL={MIN,MAX,EMERG,ALERT,CRIT,ERR,WARNING,NOTICE,INFO,DEBUG}
+LF_CBCMAC={OPENSSL,AESNI} //NOTE that AESNI is only available on x86 systems with AES assembly instructions. If available AESNI is more performant.
```

Comment:
A bit too much text for this list. You could add a shorter version of this remark to the description of the option in the CMake file.

```suggestion
LF_CBCMAC={OPENSSL,AESNI}
```

Reply:
Done